packages/redux/src/products/index.ts

```diff
@@ -4,6 +4,8 @@ import serverInitialState from './serverInitialState';
 
 export * from './actions';
 export * from './selectors';
+export * from './types';
+export * from './utils';
 
 export { actionTypes, entitiesMapper, serverInitialState };
```

Comment:
Forgot that these should be exported with a different name, like this:

```suggestion
export { actionTypes as actionTypesProducts, entitiesMapperProducts, serverInitialStateProducts };
```